### ISSUE TITLE
Hiding the quantity

### ISSUE THREAD

**Issue by rasgo-cc** [broofa/BroofaJS#9]
This is a feature request: 
I'd like to be able to get only the singular/pluralized word, without the quantity.

Maybe the format function could be used for that? Something like this:
`simplur`${[1, ()=>null]}unit[|s]`

Currently, returning `null` or an empty string has the same effect (the quantity is printed as is).

My use-case: I'm using simplur with React where I might want to apply different css classes to either the quantity value and the pluralized string. Ex: `<span className="qty">{qty}</span> <span className="word">unit(s)</span>` 

PS: Love the approach of this library.

**Comment by broofa** [comment]
You can provide multiple tokens for a quantity:

```
> simplur`<span className="qty">${1}</span> <span className="word[|s]">unit[|s]</span>`
'<span className="qty">1</span> <span className="word">unit</span>'
> simplur`<span className="qty">${2}</span> <span className="word[|s]">unit[|s]</span>`
'<span className="qty">2</span> <span className="words">units</span>'
```

Would this work for your use case?

**Comment by rasgo-cc** [comment]
Even though it works, it's not ideal as it requires to render the HTML from a string which is the front-end world is considered a dangerous practice (prone to injection).  

Other use case for this would be in this kind of sentence where the quantity is also implicit:
"Are you sure you want to remove this/these item/items?"

**Comment by rasgo-cc** [comment]
I just created a PR with the changes I requested #10 

In my opinion, this makes the `formatQuantity` function more predictable, as the returned value is *always* what goes into the output stream. So if "" or null or undefined is returned, nothing will appear in the output string (quantity omitted).

**Comment by broofa** [comment]
FYI, I reworked the code a bit and ended up adding whitespace removal back in.  It works slightly differently though: If a quantity is hidden, only the whitespace immediately following the quantity is removed.  Published in simplur@3.0.0
